src/sentry/static/sentry/app/actionCreators/incident.jsx

```diff
@@ -13,7 +13,13 @@ import {t} from 'app/locale';
  * @param {String} title Title of the incident
  * @param {String[]} groups List of group ids
  */
-export async function createIncident(api, organization, title, groups) {
+export async function createIncident(
+  api,
+  organization,
+  title,
+  groups,
+  dateStarted = new Date()
```

Comment:
Does Javascript call this default each time the function is called, or is it like python where it is called once during import time?

Reply:
Each time the function is called. This gets transpiled so that arguments are read within the function body.